modules/aws_asg/cloud-config.yml.tpl

```diff
@@ -57,53 +57,45 @@ write_files:
 - content: '${base64encode(syslog_address)}'
   encoding: b64
   path: /var/tmp/travis-run.d/syslog-address
-- content: '${base64encode(file("${here}/check-unregister-netdevice.bash"))}'
+- content: '${base64encode(file("${assets}/travis-worker/check-unregister-netdevice.bash"))}'
   encoding: b64
   owner: 'root:root'
   path: /var/tmp/travis-run.d/check-unregister-netdevice
   permissions: '0750'
-- content: '${base64encode(file("${assets}/travis-worker/travis-worker.conf"))}'
-  encoding: b64
-  owner: 'root:root'
-  path: /var/tmp/travis-worker.conf
-- content: '${base64encode(file("${here}/check-docker-health.bash"))}'
+- content: '${base64encode(file("${assets}/travis-worker/check-docker-health.bash"))}'
   encoding: b64
   owner: 'root:root'
   path: /var/tmp/travis-run.d/check-docker-health
   permissions: '0750'
-- content: '${base64encode(file("${here}/check-docker-health-crontab"))}'
+- content: '${base64encode(file("${assets}/travis-worker/check-docker-health.crontab"))}'
   encoding: b64
   owner: 'root:root'
-  path: /etc/cron.d/check-docker-health-crontab
+  path: /etc/cron.d/travis-worker-check-docker-health
   permissions: '0644'
-- content: '${base64encode(file("${here}/kill-old-containers.bash"))}'
+- content: '${base64encode(file("${assets}/travis-worker/clean-up-containers.bash"))}'
   encoding: b64
   owner: 'root:root'
-  path: /var/tmp/travis-run.d/kill-old-containers
+  path: /var/tmp/travis-run.d/clean-up-containers
   permissions: '0750'
-- content: '${base64encode(file("${here}/kill-old-containers-crontab"))}'
+- content: '${base64encode(file("${assets}/travis-worker/clean-up-containers.crontab"))}'
   encoding: b64
   owner: 'root:root'
-  path: /etc/cron.d/kill-old-containers-crontab
+  path: /etc/cron.d/travis-worker-clean-up-containers
   permissions: '0644'
-- content: '${base64encode(file("${here}/high-cpu-check.bash"))}'
+- content: '${base64encode(file("${assets}/travis-worker/high-cpu-check.bash"))}'
   encoding: b64
   owner: 'root:root'
   path: /var/tmp/travis-run.d/high-cpu-check
   permissions: '0750'
-- content: '${base64encode(file("${here}/high-cpu-check-crontab"))}'
+- content: '${base64encode(file("${assets}/travis-worker/high-cpu-check.crontab"))}'
```

Comment:
Any reason not to just use the same filename in both places (i.e. in the repo and on the host)?

Reply:
For me, at least, the `.crontab` extension enables syntax highlighting, which I was assuming at least one other person also appreciated :grin: